an-apluss/AutoMart
# #166202826 Write test suite for user sign up and sign in endpoints

#### What does this PR do?
- Write tests for the POST `/api/v1/auth/signup` and `/api/v1/auth/signin` endpoints

#### Description of Task to be completed?
- install and setup necessary test tools
- write test cases for user sign up endpoint
- write test cases for user sign in endpoint

#### How should this be manually tested?
- Clone the repository using git clone `https://github.com/an-apluss/AutoMart.git` . in your terminal
- run `npm install` on your terminal
- run `npm run start` to start up the server
- run `npm test` to see test results
- delivers[#166202826]

#### Any background context you want to provide?
Create environment variables `PORT=portnumber` 

#### What are the relevant pivotal tracker stories?
- PT Story link - ( [166202826](https://www.pivotaltracker.com/story/show/166202826) )

#### Screenshots(if appropriate)
-N/A
